queue_entry.py

```diff
@@ -26,7 +26,6 @@
 import time
 import queue_model_objects_v1 as queue_model_objects
 import os
-import ShapeHistory as shape_history
```

Comment:
OK. I will do it soon.

From: Matias Guijarro [mailto:notifications@github.com]
Sent: Monday, February 01, 2016 3:02 PM
To: mxcube/HardwareObjects
Cc: Jie Nan
Subject: Re: [HardwareObjects] a few fixes of mockups (#9)


In queue_entry.py<https://github.com/mxcube/HardwareObjects/pull/9#discussion_r51420636>:

> @@ -26,7 +26,6 @@

>  import time

>  import queue_model_objects_v1 as queue_model_objects

>  import os

> -import ShapeHistory as shape_history

@JieNanMAXIV<https://github.com/JieNanMAXIV> are you going to push another commit to fix this to this PR? So I can merge it

—
Reply to this email directly or view it on GitHub<https://github.com/mxcube/HardwareObjects/pull/9/files#r51420636>.